src/locales/en.json

```diff
@@ -4,67 +4,67 @@
     "about": "About",
     "spreadsheet": "Spreadsheet",
     "contactus": "Contact Us",
-    "whitepaper": "Whitepaper"
+    "whitepaper": "White Paper"
   },
   "current_lang_two_letter_code": "en",
   "calculator": {
     "precautions": {
       "label": "Precautions",
       "environment_header": "Environment",
       "environment_question": "Is it indoor or outdoor?",
-      "no_intimate_bonus_outdoors": "Due to very close distances, we are not confident that being outdoors reduces the risk in a substantial way. Thus, we are not providing any bonus for being outdoors when intimate.",
+      "no_intimate_bonus_outdoors": "Due to very close distances, being outdoors probably doesn't reduce the risk substantially. There is no bonus for being outdoors when intimate.",
```

Comment:
Can you help me understand what you prefer about this? It sounds more authoritative, and removes the author's perspective, but I think we would prefer to keep it clear that this is a judgment call we are making based on minimal data.

Reply:
It is so wishy washy. "we" and "that" is redundant right off the bat. It is a garden-path sentence, where reading the next sentence is necessary to understand it.

Reply:
This wording is still quite verbose qualified; if you wanted to be concise why not just "Being outdoors doesn't reduce risk when intimate"